dist/percent-circle-directive.js

```diff
@@ -55,13 +58,14 @@
 
 				// takes the percentage and returns complimentary degrees
 				function getDegrees(percent) {
+					if (percent > 100) percent = 100;
 					return (percent/100) * 360;
 				}
 
 				// Loop through and increment percentage to animate in DOM
 				function animatePercentChange(fromVal, toVal) {
 					if(fromVal != toVal) {
-						$scope.curPercent = (fromVal < toVal) ? fromVal + 1 : fromVal - 1;
+						$scope.curPercent = (fromVal < toVal) ? fromVal + $scope.step : toVal;
```

Comment:
This is wrong. The point of the ternary is to allow increasing number animation on the left side of the colon and decreasing number animation on the right side of the colon. You have removed the decreasing number animation. It should be `$scope.curPercent = (fromVal < toVal) ? fromVal + $scope.step : fromVal - $scope.step;`